README.md

```diff
@@ -56,11 +56,25 @@ _Note_: To contribute, send a pull request to this repository. Note that this re
 ### Development Platform
 
 - [ONNX Runtime](https://onnxruntime.ai/): Platform agnostic model runtime to use ML models.
+- [TensorFlow](https://www.tensorflow.org/): An end-to-end open source platform for machine learning.
+- [PyTorch](https://pytorch.org/): An open source machine learning framework that accelerates the path from research prototyping to production deployment.
+- [PyTorch Lightning](https://github.com/Lightning-AI/pytorch-lightning) Pretrain, finetune and deploy AI models on multiple GPUs, TPUs with zero code changes.
+- [Scikit-learn](https://scikit-learn.org/stable/): A simple and efficient tools for predictive data analysis.
+- [NCNN](https://github.com/Tencent/ncnn): A high-performance neural network inference framework optimized for the mobile platform.
+- [JAX](https://github.com/google/jax): Composable transformations of Python+NumPy programs: differentiate, vectorize, JIT to GPU/TPU, and more!
+- [Keras](https://keras.io/): A high-level neural networks API, written in Python and capable of running on top of TensorFlow, CNTK, or Theano.
+- [Keras Core](https://github.com/keras-team/keras-core): A multi-backend implementation of the Keras API, with support for TensorFlow, JAX, and PyTorch.
+
+- [NumPy](https://numpy.org/): A fundamental package for scientific computing with Python.
+- [Polars](https://github.com/pola-rs/polars): A blazingly fast DataFrame library implemented in Rust & Python.
+- [Matplotlib](https://matplotlib.org/): A comprehensive library for creating static, animated, and interactive visualizations in Python.
 
 ### Web
 
 - [Transformers.js](https://huggingface.co/docs/transformers.js/en/index): A library to run cutting edge models directly in-browser.
 - [huggingface.js](https://huggingface.co/docs/huggingface.js/en/index): A library to play with models on Hugging Face Hub through javascript.
+- [TensorFlow.js](https://www.tensorflow.org/js): A library for machine learning in JavaScript.
+- [Mediapipe](https://developers.google.com/mediapipe/api/solutions/js) A framework that has prebuilt and customizable ML solutions, ready to deploy on Web
```

Comment:
I think if it can be used for both mobile and web we could move it up in development platform part under ONNX

Reply:
ah sorry I meant Mediapipe specifically not TFJS :) thank you, it works then